### ISSUE TITLE
can not close/mention issues from pull-requests or pull-requests from issues

### ISSUE THREAD

**Issue by MorphBonehunter** [go-gitea/gitea#999]
- Gitea version (or commit ref): 1.0.1
- Git version: 2.11.1
- Operating system: Arch Linux
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No - try.gitea.io seems broken at the moment, UI returns mostly http 500 and git says:
```
remote: 2017/02/21 13:56:36 [...s/setting/setting.go:1030 NewXORMLogService()] [E] Unknown log mode: file
remote: error: hook declined to update refs/heads/master
```
  - [ ] Not relevant
- Log gist:

## Description
I can not close issues from pull requests mit the mentioned keywords from `models/action.go`.
If i try to use say "`fixes #1`" the issue is linked correctly but in the issue i can not see this mention.
Also if i merge the request it doesn't closes the issue and it is no entry in an pull request if i  Reference it from an issue.

**Comment by lunny** [comment]
could update to the latest commit and then test it? We just merged one commit to fix your mentioned problem on log.

**Comment by MorphBonehunter** [comment]
@lunny the mentioned log problems was not on my local install, it was on `try.gitea.io` so i can not reproduce the real issue with the mentions there at the moment.

BTW: "kind/feature" label? I think the "mentions" should work actually as the code is already there in `models/action.go`

**Comment by MorphBonehunter** [comment]
@lunny: i have to come back to this.
Maybe i'm wrong but please take a look at

https://github.com/go-gitea/gitea/blob/master/models/action.go#L49

i think this should be flagged as bug and not as feature.

**Comment by lunny** [comment]
@MorphBonehunter 
```
remote: 2017/02/21 13:56:36 [...s/setting/setting.go:1030 NewXORMLogService()] [E] Unknown log mode: file
remote: error: hook declined to update refs/heads/master
```
That issue should be fixed on latest master version. 
So this should be another one.

**Comment by MorphBonehunter** [comment]
@lunny This was the error i got when i tried this to reproduce on **try.gittea.io** not on my local Install.
I don't think it's related.

**Comment by lunny** [comment]
Since we cannot reproduce this again. I will close this one. Please feel free to reopen it if you have further information.

**Comment by MorphBonehunter** [comment]
@lunny: I tried it again on https://try.gitea.io/ and it seems fixed in 1.1...so thanks
i still running 1.0.1 as i'm still waiting on 1.1.1 for upgrading.
Seems that milestone was last touched 3 weeks ago are there any plans to release 1.1.1 in the near future?

**Comment by lunny** [comment]
I think we will ASAP
